### PR TITLE
Fix y2log path in popup messages

### PR DESCRIPTION
**Pull request by scabrero**
## Problem

The popup messages refer to /var/log/YaST/y2log when it should be /var/log/YaST2/y2log.

- https://bugzilla.suse.com/show_bug.cgi?id=1207831

### CODE REVIEW

File: src/lib/authserver/ui/new_dir_inst.rb [hunk 1]
```diff
@@ -120,7 +120,7 @@ def ok_handler
     # Always remove the ini file
     DS389.remove_setup_ini
     if !ok
-      Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST/y2log'))
+      Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST2/y2log'))
```
Comment (dgdavid) [inline src/lib/authserver/ui/new_dir_inst.rb:123]: You could interpolate [`Yast::Directory.logdir`](https://github.com/yast/yast-yast2/blob/7202cd8968a946879ea4df0ae071049d4f46f179/library/general/src/modules/Directory.rb#L130) instead

```suggestion
      Popup.Error(
         # TRANSLATORS: %{y2log_path} is replaced by the YaST2 logs path.
         format(
          _('Failed to set up new instance! Log output may be found in %{y2log_path}'),
          y2logpath: Yast::Directory.logdir
        )
      )
```

**Review by dgdavid — APPROVED** [review]
LGTM, thanks.

Just in case you want, I left a comment illustrating how to print the path using Yast::Directory.logdir.

Approved anyway.

File: src/lib/authserver/ui/new_dir_inst.rb (hunk 1 above)
Comment (scabrero) [reply src/lib/authserver/ui/new_dir_inst.rb]: Thanks, I implemented your suggestion.

**Comment by dmulder** [comment]
LGTM.

**Pull request merged**